### PR TITLE
Autentication github

### PR DESCRIPTION
**Pull request by Ktagudelo**

### CODE REVIEW

**Comment by Ktagudelo** [comment]
No se porque fallo los test ya que en local funciona correctamente y el archivo esta configurado como CRLF
image
como lo puedo solucionar gracias

File: .eslintrc.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  'comma-dangle': ['error', 'never']
+  'linebreak-style': ['error', 'windows']
```
Comment (veronicatofino) [inline .eslintrc.yml]: Don't configure it to use windows

**Comment by veronicatofino** [comment]
@Ktagudelo Is failing because Github only accepts LF lines that are the UNIX for LINUX configuration. But in your local you configure it with CRLF that is only taken by windows, that's why is failing only in Github Actions. Change your local files to use LF 😃

**Comment by Ktagudelo** [comment]
@veronicatofino Thank you very much, the tests are ok

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "semi":true,
+    "singleQuote": true,
+    "tabWidth": 2,
+    "useTabs": false,
+    "trailingComma": "all"
+}
```
Comment (holgiosalos) [inline .prettierrc.json]: EOF

File: test/GithubApi.Authentication.test.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    it('Via OAuth2 Tokens by parameter', async () => {
+      const response = await axios.get(
+        `${urlBase}/repos/${githubUserName}/${repository}`,
+        { access_token: process.env.ACCESS_TOKEN }
+      );
+
+      expect(response.status).to.equal(StatusCodes.OK);
+      expect(response.data.description).equal(
+        'This is a Workshop about Api Testing in JavaScript'
+
```
Comment (holgiosalos) [inline test/GithubApi.Authentication.test.js]: remove this blank line

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**